### ISSUE TITLE
Setting a Makefile Path is expecting a path and filename, but should only need a path.

### ISSUE THREAD

**Issue by KamilPiekutowski** [microsoft/vscode-makefile-tools#365]
When specifying a path, the script is checking if the provided path is a file in the code snipped here:

```js
function checkFileExistsSync(filePath) {
    try {
        return fs.statSync(filePath).isFile();
    }
    catch (e) {
    }
    return false;
}```

It is a little confusing for the first time users to figure out either settings need only a path, or path/Makefile.

**Comment by bobbrow** [comment]
Thank you for the feedback.  I think it would make sense for us to append `/Makefile` to the path if we detect that it's a directory and try the `isFile` query again.

**Comment by KamilPiekutowski** [comment]
I can implement a fix and create PR, if other contributors are too busy. However, thank you for a quick response.

**Comment by andreeis** [comment]
@KamilPiekutowski , a fix for this will be released with 0.7 next week but if you would like to see how this works earlier you can install one vsix from [this link](https://github.com/microsoft/vscode-makefile-tools/actions/runs/4451312188).

go to the Extensions panel on the left
click "..."
run "Install from vsix" from the context menu

Let us know if you encounter any other issues or if you have any other suggestions/recommendations in this area of functionality.

**Comment by gcampbell-msft** [comment]
Completed in 0.7.0. Closing! 👍
